src/widgets/AButton.h

```diff
@@ -13,36 +13,50 @@
 #define __AUDACITY_BUTTON__
 
 #include <vector>
+#include <array>
+#include <memory>
 
 #include <wx/setup.h> // for wxUSE_* macros
 #include <wx/window.h> // to inherit
 
-#include "ImageRoll.h" // member variable
```

Comment:
This deletion requires you insert the `#include` in `ExpandingToolBar.cpp` so that the commit "AButton refactoring" builds